src/libs/actions/ReimbursementAccount/validateBankAccount.js

```diff
@@ -11,7 +11,7 @@ import * as errors from './errors';
  * @param {String} validateCode
  */
 function validateBankAccount(bankAccountID, validateCode) {
```

Comment:
We are changing this function [here](https://github.com/Expensify/App/pull/10838/files#diff-66ce9adc44f2a3a4b12481610a8bd2827bc7cb71c4d3347f0628521cb9129a1bL415-L420) and this file will no longer exist. The loading/error state changes will be handled in the new API command.

Reply:
Should this PR be put on hold until the change is merged?

Reply:
Sounds good. I'll push the rest of the requested changes and wait til the PR is merged.

Reply:
FYI [this PR](https://github.com/Expensify/App/pull/10838) is merged